src/main/scala/com/microsoft/hyperspace/actions/RefreshAppendAction.scala

```diff
@@ -57,7 +57,7 @@ class RefreshAppendAction(
     super.validate()
 
     if (appendedFiles.isEmpty) {
-      throw HyperspaceException("Refresh aborted as no appended source data files found.")
+      throw NoChangesDetected("Refresh aborted as no appended source data files found.")
```

Comment:
Let's update this to `Refresh append aborted ...` so that there is no confusion now that we are running two actions.

Reply:
thanks @imback82 , fixed